index.js

```diff
@@ -1 +1,5 @@
-console.log('hello world');
+const array = [2, 3, 5, 6, 1];
+const sortedArray = array.sort((item1, item2) => item1 > item2);
+
+console.log(sortedArray);
+console.log('sorted');
```

Comment:
instead of 2 console.log you can do it in one
`console.log('Here sorted array: ' + sortedArray.toString());`